### PR TITLE
[RHACS] Release Notes for 3.74.4 patch

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Includes release notes update for patch release 3.74.4
Preview: https://60642--docspreview.netlify.app/openshift-acs/latest/release_notes/374-release-notes.html#resolved-in-version-3744

For https://issues.redhat.com/browse/ROX-17168

No cherrypicks required.

> **NOTE**
> Do not merge yet. It will release on 2023-06-05

### CODE REVIEW

File: release_notes/374-release-notes.adoc [hunk 1]
```diff
@@ -16,6 +16,7 @@ toc::[]
 |`3.74.1` | 20 March 2023
 |`3.74.2` | 13 April 2023
 |`3.74.3` | 2 May 2023
+|`3.74.3` | 31 May 2023
```
Comment (tommartensen) [inline release_notes/374-release-notes.adoc:19]:
```suggestion
|`3.74.4` | 31 May 2023
```

**Review by tommartensen — COMMENTED** [review]
small typo

File: release_notes/374-release-notes.adoc [hunk 2]
```diff
@@ -379,6 +380,16 @@ Release date: 2 May 2023
 * This release of {product-title-short} includes a fix for link:https://access.redhat.com/security/cve/CVE-2023-28617[CVE-2023-28617] for {op-system-base}.
 * This release includes a fix for an issue with migration to the PostgreSQL database (Technology Preview).
 
+[id="resolved-in-version-3744"]
+=== Resolved in version 3.74.4
+
+Release date: 31 May 2023
+
+* {product-title-short} 3.74.4 is built with updated golang to fix the following CVEs:
```
Comment (agantony) [inline release_notes/374-release-notes.adoc:388]:
```suggestion
* {product-title-short} 3.74.4 is built with updated Golang to fix the following CVEs:
```

File: release_notes/374-release-notes.adoc [hunk 3]
```diff
@@ -379,6 +380,16 @@ Release date: 2 May 2023
...
+** link:https://access.redhat.com/security/cve/cve-2023-24540[CVE-2023-24540]
+** link:https://nvd.nist.gov/vuln/detail/cve-2023-24539[CVE-2023-24539]
+** link:https://nvd.nist.gov/vuln/detail/cve-2023-29400[CVE-2023-29400]
```
Comment (agantony) [inline release_notes/374-release-notes.adoc:391]: s/ Maybe you could write a one-liner about "what is being fixed" 
eg: CVE-2023-24540: Fixed .......
Same for the other two, it is up to you though!

File: release_notes/374-release-notes.adoc [hunk 4]
```diff
@@ -379,6 +380,16 @@ Release date: 2 May 2023
 * This release of {product-title-short} includes a fix for link:https://access.redhat.com/security/cve/CVE-2023-28617[CVE-2023-28617] for {op-system-base}.
 * This release includes a fix for an issue with migration to the PostgreSQL database (Technology Preview).
 
+[id="resolved-in-version-3744"]
+=== Resolved in version 3.74.4
+
+Release date: 31 May 2023
```
Comment (agantony) [inline release_notes/374-release-notes.adoc:386]: 5th Jun?

File: release_notes/374-release-notes.adoc [hunk 5]
```diff
@@ -16,6 +16,7 @@ toc::[]
 |`3.74.1` | 20 March 2023
 |`3.74.2` | 13 April 2023
 |`3.74.3` | 2 May 2023
+|`3.74.4` | 31 May 2023
```
Comment (agantony) [inline release_notes/374-release-notes.adoc:19]: 5th Jun?

**Review by agantony — COMMENTED** [review]
Overall LGTM, added only a few small suggestions!

File: release_notes/374-release-notes.adoc (hunk 3 above)
Comment (gaurav-nelson) [reply release_notes/374-release-notes.adoc:391]: Thank you. That's what we have in Errata and other release notes, so I'm keeping it as it is.

**Pull request merged**